File: exercices/cpp/ex003.cpp
// ex001 - Crie um programa que escreva "Olá Mundo!" na tela.

#include <iostream>
using namespace std;
int main()
{
  int n1, n2;
  cout << "Digite um número: ";
  cin >> n1;
  cout << "Digite outro: ";
  cin >> n2;
  int s = n1 + n2;
  cout << "A soma entre " << n1 << " e " << n2 << " resulta em " << s << "." << endl;
}

File: exercices/java/ex003.java
// ex003 - Crie um programa que leia dois números e mostr a soma entre eles.

import java.util.Scanner;

public class ex003{
  public static void main(String[] args){
    Scanner read = new Scanner(System.in);
    System.out.print("Digite um número: ");
    int n1 = read.nextInt();
    System.out.print("Digite outro: ");
    int n2 = read.nextInt();
    int s = n1 + n2;
    System.out.println("A soma entre " + n1 + " e " + n2 + " resulta em " + s +".");
  }
}